test/cases/0016-aliases

```diff
@@ -0,0 +1,34 @@
+#! /usr/bin/env janetsh
+
```

Comment:
A test case executing a mutiple word alias command with multiple args would be nice to have.